src/components/editors.tsx

```diff
@@ -115,6 +104,15 @@ export const DocumentEditor: React.FC<IProps> = ({
     );
     const { isAllowedToCreateDocuments, isAllowedToEditDocuments } =
         useUserPermissions();
+    const isLoadingEditor =
```

Comment:
show the loader if the editor and document do exist but the editor is not the correct editor for the document